java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/RemoteSePluginImpl.java

```diff
@@ -39,26 +37,32 @@
     public static final String DEFAULT_PLUGIN_NAME = "RemoteSePlugin";
 
     // in milliseconds, throw an exception if slave hasn't answer during this time
-    public final long rpc_timeout;
-
-    // private final VirtualReaderSessionFactory sessionManager;
+    public final long rpcTimeout;
 
     private final VirtualReaderSessionFactory sessionManager;
     protected final DtoSender dtoSender;
     private final Map<String, String> parameters;
 
+    /* The observers of this object */
+    private Set<ObservablePlugin.PluginObserver> observers;
```

Comment:
observers private attribute: the observers can't be managed though a 'Set', but only though a 'List'. Because the observers are unknown objects. A Set collection requires that the objects implements the methods 'equals': impossible if the objects are unknown.